plugins/catalog-graph/src/components/CatalogGraphCard/CatalogGraphCard.tsx

```diff
@@ -55,28 +55,25 @@ const useStyles = makeStyles<Theme, { height: number | undefined }>(
   { name: 'PluginCatalogGraphCatalogGraphCard' },
 );
 
-export const CatalogGraphCard = (props: {
-  variant?: InfoCardVariants;
-  relationPairs?: RelationPairs;
-  maxDepth?: number;
-  unidirectional?: boolean;
-  mergeRelations?: boolean;
-  kinds?: string[];
-  relations?: string[];
-  direction?: Direction;
-  height?: number;
-  title?: string;
-  zoom?: 'enabled' | 'disabled' | 'enable-on-click';
-}) => {
+export const CatalogGraphCard = (
+  props: Omit<
+    EntityRelationsGraphProps,
+    'rootEntityNames' | 'onNodeClick' | 'className'
```

Comment:
Why omit these? Maybe it would make sense to put them in a `cardProps` prop instead, and deprecate the old values?

Reply:
We override them in the Component itself, I can make them optional in the carProps prop as overrides to functionality in the card itself?

Reply:
It's a little unfortunate that we're picking them out and not letting configuration of them in the first place, it's a little side effect of not having all the options available I guess. I think it seems fine to allow these to be overridden but provide the defaults that are currently being set in the component?  What do you think?

Reply:
Makes sense, updated it to reflect that.